Source/IInterceptStrategy.cs

```diff
@@ -28,21 +28,26 @@ internal interface IInterceptStrategy
 
 	internal class InterceptorContext
 	{
-		private Dictionary<string, List<Delegate>> invocationLists = new Dictionary<string, List<Delegate>>();
-		private List<ICallContext> actualInvocations = new List<ICallContext>();
-		private List<IProxyCall> orderedCalls = new List<IProxyCall>();
+		private readonly Dictionary<string, List<Delegate>> invocationLists = new Dictionary<string, List<Delegate>>();
+		private readonly List<ICallContext> actualInvocations = new List<ICallContext>();
+		private readonly List<IProxyCall> orderedCalls = new List<IProxyCall>();
 
-		public InterceptorContext(Mock Mock, Type targetType, MockBehavior behavior)
+		public InterceptorContext(Mock mock, Type targetType)
 		{
-			this.Behavior = behavior;
-			this.Mock = Mock;
+			this.Mock = mock;
 			this.TargetType = targetType;
 		}
-		public Mock Mock { get; private set; }
-		public Type TargetType { get; private set; }
-		public MockBehavior Behavior { get; private set; }
 		
-		#region InvocationLists
+        public Mock Mock { get; private set; }
```

Comment:
This is all noise :(. I have provided a .editorconfig file that will be picked up by the EditorConfig extension in VS (available from the VS gallery) to make this easier.